docs/elements.md

```diff
@@ -2317,10 +2317,10 @@ Displays the contents of question directions.
 
 #### Details
 
-Contents are only shown during question input portion. When a student
-either makes a submission or receives the correct answer, the information
-between these tags is hidden. If content exists outside of a question panel,
-then it will be displayed alongside the answer.
+Generally, any text placed in `question.html` will appear in
+`question-panel`, `submission-panel`, and `answer-panel`. To prevent
+content from being displayed in the `submission-panel` and
+`answer-panel`, surround that content with the `<pl-question-panel>` tags.
```

Comment:
Two nits:

- I don't love "text" as it excludes things like PL and HTML elements. Hence, "contents", though bikeshed if you'd like!
- We don't ever refer to `question-panel`, etc. - it's always "the question panel" and so on.

```suggestion
Generally, the contents of `question.html` will appear in
the question panel`, submission panel, and answer panel. To prevent
content from being displayed in the submission panel and
 answer panel, surround that content with the `<pl-question-panel>` tags.
```